### ISSUE TITLE
(5) Create new A-Z List of Collections page: Desktop

### ISSUE THREAD

**Issue by JessWaggs** [UCSCLibrary/dams_project_mgmt#277]
### Descriptive Summary

As an end user, I want to view a complete list of all collections included in the DAMS.

Create new page displaying all DAMS collections, sorted alphabetically:
![AtoZ_Page_Page_1](https://user-images.githubusercontent.com/6412857/81118781-5f3df000-8ede-11ea-8d9d-758e7ae5539d.png)
 
### Acceptance Criteria

This is what done looks like

* [ ] Box with a brief description of the DAMS is included at the top
* [ ] Thumbnails for each collection are displayed
* [ ] The title of each collection is linked to the collection's landing page
* [ ] Collection title's are pulled from a collection's Title element
* [ ] Collection's descriptions are pulled from a collection's Description element
* [ ] Include link to A-Z page on DAMS homepage (refer to UCSCLibrary/dams_project_mgmt#279, UCSCLibrary/dams_project_mgmt#280 for homepage mockups)

### Related work
Mock up ticket: UCSCLibrary/dams_project_mgmt#247 
Mobile ticket: UCSCLibrary/ucsc-library-digital-collections#375 
[Mock up in Google Drive](https://drive.google.com/open?id=1LZCYHTut1TaeU-eGGNOHDNyGxo_BgMtm)

UCSCLibrary/dams_project_mgmt#366 and UCSCLibrary/dams_project_mgmt#324 are prerequisites

**Comment by JessWaggs** [comment]
@rmjaffe Can you check my work on which fields the metadata for title and description are pulled from?

**Comment by rmjaffe** [comment]
@JessWaggs Perfect!

**Comment by rschwab** [comment]
@JessWaggs Few questions:
1. What should the text of the "Our Collections" box be? This will need to be figured out before the ticket can be completed.
2. Should the A-Z order go row first (left to right, then down), or in columns first?
3. Should the collection description text be truncated, and if so what limit?

**Comment by JessWaggs** [comment]
@rschwab 

1) @rmjaffe - Can you check with Spec Coll on this? I'm thinking something short that just explains what digital collections are and what users can expect to find would be ideal

2) Let's go left to right

3) Can we truncate text so that it doesn't go beyond the height of the thumbnails?

**Comment by rmjaffe** [comment]
@tmariamora @katedundon:  May I ask you two come up with a general description of our digital collections?  As Jess said above: something short that just explains what digital collections are and what users can expect to find would be ideal.  This language will display in an 'our collections' box at the top of the A-Z collection page in the DAMS.  There is a mockup above in the work description.

I recalled that one point we'd had language on our Cdm landing page describing our digital holdings, but it looks like that's gone.  I also looked to see if there was anything on our [digital collections libguide](https://guides.library.ucsc.edu/digitalcollections) and there is not.  Perhaps whatever is drafted and added to the A-Z page should also be added to the libguide?

**Comment by tmariamora** [comment]
Sure. Can we get a deadline? This week is crazy for me ;)

**Comment by rmjaffe** [comment]
@rschwab @JessWaggs When do you need the digital collection text by?

**Comment by rschwab** [comment]
I'm just starting work on this ticket, so its not blocking me at all. Once my work is done we will need this text before this page can go into production.

**Comment by rmjaffe** [comment]
@tmariamora Our current sprint ends next Wednesday, 2/9 but I'm OK with delaying publication of the page in order to give you the time you need to draft something.  If a deadline helps, would next Friday 2/12 do?

**Comment by tmariamora** [comment]
That should work. Thanks!

**Comment by katedundon** [comment]
I've drafted a collections description in this document. Feel free to suggest changes if you think it needs wordsmithing.

[About the UCSC Library Digital Collections](https://docs.google.com/document/d/1RVezVzEKRLrwBss-5m7ESC3gYESg_-1J04QENcfDe_U/edit?ts=601da5b7)

**Comment by rmjaffe** [comment]
@JessWaggs could you have a look at this on staging?

**Comment by rmjaffe** [comment]
@snehagunduraoUL the A-Z page looks great to me.  Thank you!  I've asked @JessWaggs to have look and to sign off on it.

**Comment by rmjaffe** [comment]
Oops, I did just bump into something.  At the foot of the A-Z page where you can page through the pages of content, when clicking on '2' or 'Next' I got an error message:
![Screen Shot 2021-11-29 at 4 14 58 PM](https://user-images.githubusercontent.com/32003407/143962696-03d06133-ce04-459d-b83d-c53ec39ddc66.png)

![Screen Shot 2021-11-29 at 4 15 51 PM](https://user-images.githubusercontent.com/32003407/143962751-b2ba32f2-eb4e-4d08-b299-b47eb009c66a.png)

Are all the collection 'results' on Page 1?  Is there no Page 2?  Should there be a Page 2 or do we want every collection to appear on a single page, if so should we remove that pager tool from the bottom?

**Comment by JessWaggs** [comment]
There seems to be a little wonkiness with the layout
![Screen Shot 2021-11-29 at 4 41 02 PM](https://user-images.githubusercontent.com/6412857/143964852-2a2cce00-beb4-410a-baf9-346ce9223e21.png)

I think this iteration can move forward. Long term, I wonder if we can do two things:
1) Make the thumbnails the same size/shape (like the homepage)
2) Have fewer collections listed on each page (maybe 10?)

**Comment by snehagunduraoUL** [comment]
Thanks for the review @rmjaffe and @JessWaggs. The pagination does not work anywhere on staging as far as I have observed. Can we please open a separate issue for that? You can assign that to me.

**Comment by snehagunduraoUL** [comment]
@JessWaggs Sure. Will do.

**Comment by rmjaffe** [comment]
@snehagunduraoUL Yes, I can write up a ticket regarding the pagination tomorrow.  Are the presentation issues that Jess mentioned, conformity of the thumbnails and the number of items per page, unique to staging or will thing look the same in production.  As a note for our conversation about the staging site tomorrow, in reviewing other pieces of work I know that things in the current iteration of staging aren't always how they will appear in production.

**Comment by snehagunduraoUL** [comment]
@rmjaffe presentation issues looks the same on production too. I can make the two changes that Jess has mentioned.
